courses/machine_learning/deepdive2/structured/labs/5a_train_keras_ai_platform_babyweight.ipynb

```diff
@@ -0,0 +1,749 @@
+{
```

Comment:
<p>so setting train_examples=2000000 would be training for 2x10^9 examples since train_examples is represented in thousands?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/674/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>Ya, that is a comment left over from Lak. Those examples are wrapped in a batch though so there aren't billions of batches. He used a batch size of 350.</p>

Reply:
<p>yeah that's right. in the code <code>train_examples</code>&nbsp;represents the number of examples (in thousands) to run the training job. </p>

Reply:
<p>Yup, the students can do less when they actually do the lab (hence my 20k), but they can see what Lak did when he did 2M (2M * 1k = 2B, then divide by 350 batch size gives 5714290 steps which is what he has in the printout).</p>